nimbus/src/nimbus.idl

```diff
@@ -63,6 +63,17 @@ interface NimbusClient {
     [Throws=Error]
     sequence<EnrolledExperiment> get_active_experiments();
 
+    // Getter and setter for user's participation in all experiments.
+    // Possible values are:
+    // * `true`: user has opted in.
+    // * `false`: user has opted out
+    // * null: user has not made a choice.
```

Comment:
Do we expect we'll want to reflect this `null` state in the UX differently at all?

Reply:
For opt-in experiments this would be important, however, we're not intending to support opt-in experiments.